cmake/Modules/BuildCommon.cmake

```diff
@@ -469,18 +469,36 @@ function(add_katana_sphinx_target target_name)
   endif ()
 
   get_target_property(PYTHON_ENV_SCRIPT ${target_name} PYTHON_ENV_SCRIPT)
-
-  add_custom_target(
+  #TODO(andrew): done this way since `set(tag_option "-t external") ... sphinx-build ${tag_option}...` does not work as expected
```

Comment:
BTW, have you tried

`set(tag_option "-t;external") ... sphinx-build ${tag_option}...` or  `set(tag_options "-t" "external") ... sphinx-build ${tag_options}...`

Reply:
interesting, I'll go try that

Reply:
Worked! Thanks @ddn0